WindowsServerDocs/administration/server-core/server-core-removed-roles.md

```diff
@@ -84,3 +88,8 @@ Note that some Remote Desktop role services are included in Server Core (Connect
 - Windows TIFF IFilter (**Windows-TIFF-IFilter**)
 - Wireless LAN Service (**Wireless-Networking**)
 - XPS Viewer (**XPS-Viewer**)
+
+
+* Services indicated with an asterisk ( * ) were **added** starting with Windows Server 1803 SAC / Windows Server 2019 LTSC
+* Services indicated with 2 asterisks ( ** ) were **removed** starting with Windows Server 1803 SAC / Windows Server 2019 LTSC
+* Services indicated with 3 asterisks ( *** ) were **added** starting with Feature on Demand 1903 on Server SAC or LTSC
```

Comment:
First of all, I think it would be useful to end these sentences with a period dot. Secondly, I think it could be practical to have some sort of footnote title, but I am not sure about what the correct title should be. Maybe I am overthinking this, that it does not need a title. Let us start with the period dots first, to be on the safe side.
```suggestion
* Services indicated with an asterisk ( * ) were **added** starting with Windows Server 1803 SAC / Windows Server 2019 LTSC.
* Services indicated with 2 asterisks ( ** ) were **removed** starting with Windows Server 1803 SAC / Windows Server 2019 LTSC.
* Services indicated with 3 asterisks ( *** ) were **added** starting with Feature on Demand 1903 on Server SAC or LTSC.
```